pkg/events/parsers/data_parsers.go

```diff
@@ -8,9 +8,8 @@ import (
 	"strings"
 	"sync/atomic"
 
+	"github.com/moby/moby/pkg/parsers/kernel"
```

Comment:
The reason is mainly because of the external module issue.
However, we coupled that environment into Tracee by using the `logger` and `errfmt` packages there.
As the function only use the kernel version and not other OS information, I determined that it will be easier to use the version as a string. As I needed to parse it I thought that using such a known third-party package might be easier than trying to export many parts of the code as modules.

Reply:
@AlonZivony do you want me to move `environment` as its own module? I think we should since it's required by different projects.

Reply:
I think we do.
I think we might also want to put the logger in another module, but this way we divide the project to too many pieces.
For now I think using this external library is good enough.